packages/cli/src/index.ts

```diff
@@ -191,6 +179,10 @@ function parseCmdlineArgs() {
     help:
       "Check the status of an Opstrace instance (experimental, no promises)."
   });
+  const parserInfo = subparsers.add_parser("info", {
+    help:
+      "Get infrastructure and Opstrace cluster information (experimental, no promises)."
```

Comment:
I like that. 

Please replace "cluster" with "instance", for terminology consistency (we made that decision a while ago :shrug:).

Let's maybe say "version information" because that's all there is for now.

Reply:
Done.